requirements_venv.txt

```diff
@@ -0,0 +1,29 @@
+arrow==0.17.0
+beautifulsoup4==4.9.3
+certifi==2020.6.20
+chardet==3.0.4
+click==7.1.2
+defusedxml==0.6.0
+Flask==1.1.2
+idna==2.10
+itsdangerous==1.1.0
+Jinja2==2.11.2
+kitchen==1.2.6
+lockfile==0.12.2
+MarkupSafe==1.1.1
+munch==2.5.0
+nose==1.3.7
+openidc-client==0.6.0
+pyflakes==2.2.0
+python-dateutil==2.8.1
+python-fedora==1.0.0
+python-memcached==1.59
+python-openid==2.2.5
+python-openid-cla==1.2
+python-openid-teams==1.1
+python3-openid==3.2.0
+requests==2.24.0
+six==1.15.0
+soupsieve==2.0.1
+urllib3==1.25.10
+Werkzeug==1.0.1
```

Comment:
Out of curiosity, was there a particular reason to make an additional `requirements_venv.txt` in addition to `requirements.txt` in the root directory? It might get confusing if they get out-of-date.

Reply:
The `requirements.txt` available from the get-go describes the bare minimum versioning of the dependencies needed for this project, while the `requirements_venv.txt` talks especially about the specific version of dependencies which made their way inside the virtual environment and are frozen (for now).